test/e2e/vsphere_workload_api_flux_test.go

```diff
@@ -0,0 +1,228 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func TestVSphereMulticlusterWorkloadClusterAPIGitHubFlux(t *testing.T) {
+	vsphere := framework.NewVSphere(t)
+	managementCluster := framework.NewClusterE2ETest(
+		t, vsphere,
+	)
+	managementCluster.WithClusterConfig(
+		api.ClusterToConfigFiller(
+			api.WithKubernetesVersion(v1alpha1.Kube123),
```

Comment:
Why not use the latest Kubernetes version now which is 1.24?

Reply:
One of the workload clusters is using 1.24, so that gets tested as well. There's no problem adjusting this if there's a preference for 1.24 on the management cluster though.

Reply:
Yea let's stick to the latest for the management cluster, we can do older ones for test cases that tests running a 1.24 workload cluster on a 1.22 management cluster for example, which is a test case we should cover.